bin/update-dockerfiles.sh

```diff
@@ -83,6 +82,17 @@ update_generic_linux() {
     fi
 }
 
+update_al2023_generic_linux() {
+    CORRETTO_VERSION=$1
+    MAJOR_RELEASE=$2
+
+    jdk_version=$(echo ${CORRETTO_VERSION} | cut -d'.' -f1-3)
+    jdk_build=$(echo ${CORRETTO_VERSION} | cut -d'.' -f4)
+    corretto_version=$(echo ${CORRETTO_VERSION} | cut -d'.' -f5)
+
+    ${SED} "s/ARG version=.*/ARG version=${jdk_version}.${jdk_build}-${corretto_version}/g" ./${MAJOR_RELEASE}/jdk/al2023-generic/Dockerfile
```

Comment:
Would it be better to add an `if [[ -f ./${MAJOR_RELEASE}/jdk/al2023-generic/Dockerfile ]]` and keep it all together. I don't feel strongly about this, but slightly less likely to miss an update if the code is all together.

Reply:
Sure. Will do.